pkg/errors/errors.go

```diff
@@ -106,16 +107,20 @@ func GetErrorCode(err error) string {
 		return "Expired"
 	} else if IsRevoked(err) {
 		return "Revoked"
-	} else if IsTimeout(err) {
+	} else if IsTimeout(err) || wmi.IsTimedout(err) {
 		return "Timeout"
 	} else if IsInvalidToken(err) {
 		return "InvalidToken"
-	} else if IsUnknown(err) {
+	} else if IsUnknown(err) || wmi.IsUnknown(err) {
 		return "Unknown"
 	} else if IsDeleteFailed(err) {
 		return "Delete Failed"
 	} else if IsDeletePending(err) {
 		return "Delete Pending"
+	} else if wmi.IsInvalidType(err) {
+		return "Invalid Input"
```

Comment:
Should be "Invalid Type" here :)
You may still want an additional "else if" for IsInvalidInput(), though